packages/css/src/components/buttons.css

```diff
@@ -0,0 +1,138 @@
+@layer components {
+  /* Default button */
+  .vtmn-btn {
```

Comment:
we will have to migrate to TailwindCSS in other PR (@apply for custom css)

Reply:
Thanks! Issue created #24